src/adv.py

```diff
@@ -74,7 +74,7 @@
     print(f"\n\u001b[31mDeaths: {deaths} \n\u001b[34mEXP: {exp}/100 \n\u001b[32mLevel: {level}\u001b[37m")
```

Comment:
I challenge you to refactor this code to a more compact form. Talk to peers and see how they made their code more compact.